files/templates/additional-scrape-configs.template

```diff
@@ -1,15 +1,188 @@
+      key_file: /etc/prometheus/secrets/<SECRET_NAME>/key
+      insecure_skip_verify: false
+  tls_config:
+    ca_file: /etc/prometheus/secrets/<SECRET_NAME>/ca
+    cert_file: /etc/prometheus/secrets/<SECRET_NAME>/crt
+    key_file: /etc/prometheus/secrets/<SECRET_NAME>/key
+    insecure_skip_verify: false
+  relabel_configs:
+  - target_label: __address__
+    replacement: <API_SERVER_URL>:443
+  - source_labels: [__meta_kubernetes_node_name]
+    target_label: __metrics_path__
+    replacement: /api/v1/nodes/${1}:10250/proxy/metrics/cadvisor
+  - target_label: app
+    replacement: cadvisor
+  # Add cluster_id label.
+  - target_label: cluster_id
+    replacement: <CLUSTER_ID>
+  # Add cluster_type label.
+  - target_label: cluster_type
+    replacement: tenant_cluster
+  # Add provider label.
+  - target_label: provider
+    replacement: <PROVIDER>
+  - source_labels: [__meta_kubernetes_node_address_InternalIP]
+    target_label: ip
+  - source_labels: [__meta_kubernetes_node_label_role]
+    target_label: role
+  - source_labels: [__meta_kubernetes_node_label_role]
+    regex: null
+    target_label: role
+    replacement: worker
+  metric_relabel_configs:
+  - source_labels: [namespace]
+    regex: (kube-system|giantswarm.*|vault-exporter)
+    action: keep
+  - source_labels: [__name__]
+    regex: container_network_.*
+    action: drop
+- job_name: calico-node
+  honor_timestamps: false
+  scrape_interval: 1m
+  scrape_timeout: 1m
+  scheme: https
+  kubernetes_sd_configs:
+  - api_server: https://<API_SERVER_URL>
+    role: pod
+    tls_config:
+      ca_file: /etc/prometheus/secrets/<SECRET_NAME>/ca
+      cert_file: /etc/prometheus/secrets/<SECRET_NAME>/crt
+      key_file: /etc/prometheus/secrets/<SECRET_NAME>/key
+      insecure_skip_verify: false
+  tls_config:
+    ca_file: /etc/prometheus/secrets/<SECRET_NAME>/ca
+    cert_file: /etc/prometheus/secrets/<SECRET_NAME>/crt
+    key_file: /etc/prometheus/secrets/<SECRET_NAME>/key
+    insecure_skip_verify: false
+  relabel_configs:
+  - target_label: __address__
+    replacement: <API_SERVER_URL>:443
+  - source_labels: [__meta_kubernetes_pod_name]
+    regex: (calico-node.*)
+    target_label: __metrics_path__
+    replacement: /api/v1/namespaces/kube-system/pods/${1}:9091/proxy/metrics
+  - source_labels: [__meta_kubernetes_namespace, __meta_kubernetes_pod_name]
+    regex: kube-system;calico-node.*
+    action: keep
+  - source_labels: [__meta_kubernetes_pod_container_name]
+    target_label: app
+  - source_labels: [__meta_kubernetes_namespace]
+    target_label: namespace
+  - source_labels: [__meta_kubernetes_pod_name]
+    target_label: pod_name
+  # Add cluster_id label.
+  - target_label: cluster_id
+    replacement: <CLUSTER_ID>
+  # Add cluster_type label.
+  - target_label: cluster_type
+    replacement: tenant_cluster
+  # Add provider label.
+  - target_label: provider
+    replacement: <PROVIDER>
+- job_name: aws-node
```

Comment:
Only deploying `aws-node` for AWS would be neat too

Reply:
I need to check how but yes :) This would mean a better templating tool but it should be doable

Reply:
This is fixed :)